Database/Corrections/TBC/tbcQuestFixes.lua

```diff
@@ -1076,9 +1083,15 @@ function QuestieTBCQuestFixes:Load()
         [10896] = {
             [questKeys.extraObjectives] = {{nil, ICON_TYPE_SLAY, "Kill Rotting Forest-Ragers and Infested Root-Walkers to spawn Wood Mited", 0, {{"monster", 22307}, {"monster", 22095}}}},
         },
+        [10897] = {
+            [questKeys.requiredSkill] = {182,325},
+        },
```

Comment:
```suggestion
```

This is an Alchemy (171) quest not Herbalism (182). So there must be something else up